actiontext/lib/action_text/html_conversion.rb

```diff
@@ -1,24 +0,0 @@
-# frozen_string_literal: true
-
-module ActionText
-  module HtmlConversion
```

Comment:
Public API - we'll need a deprecation cycle. Can these wrap & delegate to the new Document setup?